content/post/altswiftui-travel/index.md

```diff
@@ -0,0 +1,84 @@
+
+![AltSwiftUI code sample](altswiftui-code-sample.png)
+
+We can create AltSwiftUI views by having structs conform to the `View` protocol. Property wrappers such as @StateOject, @State, @Binding, etc. allow the view to automatically refresh when the value of these properties change. The content of the `body` computed property (as required by the `View` protocol) will define the actual UI code. Finally, the UI code is defined in a declarative way by using Swift's function builders. If you are not familiar with SwiftUI code, you should head to ![Apple's SwiftUI tutorial](https://developer.apple.com/tutorials/swiftui) for a fast start.
+
+One of the main differences between SwiftUI and AltSwiftUI is that AltSwiftUI doesn't use associated types in views. This means that it is not necessary to use opaque return types (`some View`). 
+
+## Extended features
+
+AltSwiftUI interface is inspired by SwiftUI, but we needed more features and capabilities than originally offered, so we introduced extra functionalities in AltSwiftUI.
+
+For example, we needed deeper integrations with List and ScrollView interactions, so there are methods like `ScrollView.scrollOffset` and `List.onDragStarted`. There are also multiple small additions to some of the views and modifiers, and this list will keep growing in the future. To identify these features, you can read the source documentation during development or at the ![AltSwiftUI homepage](https://altswiftui.com).
+
+## AltSwiftUI in Rakuten Travel
+
+AltSwiftUI’s development started at the end of 2019. Our main purpose was taking on new projects while renewing the way we design software. It also allowed us to improve the speed and quality of our development, and be more future proof.
+
+We compared what SwiftUI offered with what our app needed while improving AltSwiftUI. The open source nature of this framework allowed us to meet our app expectations by adding new features.
+
+By using only AltSwiftUI to code our UI, we recently developed and published a new app titled _Rakuten Travel Premium Club_. This app is an invitation only app that provides luxurious travel destinations for booking to our costumers.
+
+### Simplified Architecture
+
+The ability to simplify our app’s architecture is a main pillar for improving our design and development. When using patterns such as MVVM or VIPER, the number of things a developer needs to worry about tend to increase. In contrast, we could transition to a MV (model view) design with AltSwiftUI at its core. 
+
+The core principle of this MV design is that developers should focus more on what they want to build, rather than how. We can delimit what is part of the UI (view layer) and what is part of our business logic/data (model layer). Because of this, we can better visualize our tasks and improve development efficiency.
```

Comment:
```suggestion
The core principle of this MV design is that developers should focus more on _what_ they want to build, rather than _how_. We can delimit what is part of the UI (view layer) and what is part of our business logic/data (model layer). Because of this, we can better visualize our tasks and improve development efficiency.
```